2022/_includes/homepage/slides.html

```diff
@@ -30,78 +30,74 @@
+                                                    </div>
+                                                    -->
 
-                                    </div>
-<!--
-                                    <div style="text-shadow: 2px -2px black; display: flex; justify-content: center;"
-                                         class='DateCountdown' data-date='{{site.data.homepage.starting_day_and_time}}'></div>
-                                    <script>
-                                        /* Set date above and uncomment below to enable countdown */
-                                        jQuery(document).ready(function ($) {
-                                            $('.DateCountdown').TimeCircles({
-                                                'animation': 'smooth',
-                                                'use_background': false,
-                                                'bg_width': 0,
-                                                'fg_width': 0,
-                                                'time': {
-                                                    'Days': {
-                                                        'text': 'Days',
-                                                        'show': true
-                                                    },
-                                                    'Hours': {
-                                                        'text': 'Hours',
-                                                        'show': true
-                                                    },
-                                                    'Minutes': {
-                                                        'text': 'Minutes',
-                                                        'show': true
-                                                    },
-                                                    'Seconds': {
-                                                        'text': 'Seconds',
-                                                        'show': true
-                                                    }
-                                                }
-                                            });
-                                        });
-                                    </script>
--->
+                                                </div>   
+                                            </td>
+                                            <td style="vertical-align: middle;">
+                                                <div class="counter-box-general-content-date" >
+                                                    <div class="counter-box-general-content-date-center">
+                                                        <div class="counter-box-general-content-date-monthyear" style="color: white">
+                                                        Rebranded to...&nbsp;&nbsp;
```

Comment:
If you care `Rebranded` has a negative connotation for some ppl, it's a marketing term after all. Why not simply "We have evolved, meet us next year at..."